cmake/inputs/patches/ep_gcssdk/build.patch

```diff
@@ -1,124 +1,119 @@
-From 9d8c327c0763229939c52fd7d3bf994f243c609f Mon Sep 17 00:00:00 2001
-From: Seth Shelnutt <Shelnutt2@gmail.com>
-Date: Sun, 7 Jun 2020 13:19:31 -0400
-Subject: [PATCH] GCS Build Patch for TileDB
+From b95493b781f372c0afbbfe0c1f8d1603513bd25a Mon Sep 17 00 : 00 : 00 2001 From
+    : Joe Maley<joe @tiledb.com>
```

Comment:
The formatting seems messed up?

Reply:
Thanks -- my pre-commit regex accidentally hit this with `clang-format`. Fixed.